app/views/content/non-uk-teachers/visas-for-non-uk-teachers.md

```diff
@@ -178,6 +177,7 @@ You’ll also have to [pay visa fees](https://www.gov.uk/visa-fees).
 
 You may need to [pay for UK healthcare](https://www.gov.uk/healthcare-immigration-application) as part of your immigration application – this is called the Immigration Health Surcharge (IHS).
 
+$teachinuk$
```

Comment:
The h2 below this has gone screwy?

![image](https://github.com/user-attachments/assets/3928a46d-f096-47ad-b305-36251f7a0ddc)